src/GraphQL.Tests/Serialization/SystemTextJson/InputsConverterTests.cs

```diff
@@ -13,7 +14,11 @@ public class InputsConverterTests
         {
             PropertyNamingPolicy = JsonNamingPolicy.CamelCase,
             WriteIndented = true,
+#if NET6_0_OR_GREATER
+            DefaultIgnoreCondition = JsonIgnoreCondition.Never,
```

Comment:
Why did this property need? In addition, I see that it is available in NET5.

Reply:
1. IgnoreNullValues is marked as obsolete. https://source.dot.net/#System.Text.Json/System/Text/Json/Serialization/JsonSerializerOptions.cs,270
2. You are right, it is available for NET 5. Changed the ifdefs.

Reply:
I think both properties may/should be omitted for ease of reading. `IgnoreNullValues` is `false` by default and `DefaultIgnoreCondition` is `JsonIgnoreCondition.Never` by default too so why to bother with all these ifdefs ?

Reply:
I tried to keep the original way and add a compatible but not obsolete way :) However I think it is safe to remove those assignments, since they are assigned to the default values anyway. If you agree, I remove them not to create confusion or obsolete warnings.

Reply:
OK to remove